### PR TITLE
[Data Factory] Adding configure factory repo operation

### PR DESCRIPTION
**Pull request by frankycrm14**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- Rest Spec PR [link](https://github.com/Azure/azure-rest-api-specs/pull/2866) Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
 		<Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-		<Version>0.7.0-preview</Version>
+		<Version>0.8.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
 		<PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
 		<PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Please update `PackageReleaseNotes` here or link it to the md file [here]( src/SDKs/DataFactory/Management.DataFactory/changelog.md) for consistency

File: src/SDKs/_metadata/datafactory_resource-manager.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      eed831212786a36c6575fd6d20d2a970dc96cfa5
+Commit:      7a6afb0f255ea0cdbcb100b9f17ff670fda64c54
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\gadongji\AppData\Roaming\npm `-- autorest@2.0.4262  
+Bootstrapper version:    C:\Users\fgutierr\AppData\Roaming\npm +-- autorest@2.0.4262  `-- oad@0.1.7   `-- autorest@1.9.1
```
Comment (dsgouda) [inline src/SDKs/_metadata/datafactory_resource-manager.txt]: Please create a generate.ps1 script similar to [this](https://github.com/frankycrm14/azure-sdk-for-net/blob/a746c7e5b1213b6ae34dbb9ef18d49457459062a/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/_metadata/datafactory_resource-manager.txt (hunk 2 above)
Comment (frankycrm14) [reply src/SDKs/_metadata/datafactory_resource-manager.txt]: done

**Comment by dsgouda** [comment]
@frankycrm14 Thanks for addressing the feedback. One minor thing, please regenerate the code using the generate.ps1 you created and commit all changes

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
 ## Current version
+## Version 0.8.0-preview
```
Comment (hvermis) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: Do not add this. I will update the version later when we release. Leave your changes under Current version

Comment (frankycrm14) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: done

**Comment by frankycrm14** [comment]
@dsgouda I ran the generate.ps1 and changes have been added to PR

**Review by hvermis — APPROVED** [review]

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
WIll merge on CIs passing

**Pull request merged**